src/tools/updateChildInvolvementLegacyTargetPopulation.js

```diff
@@ -0,0 +1,38 @@
+import { Op } from 'sequelize';
+import {
+  ActivityReport,
+} from '../models';
+import { auditLogger } from '../logger';
+
+export default async function updateChildInvolvementLegacyTargetPopulation() {
```

Comment:
this
```js
   where: {
      [Op.and]: [
        {
          imported: { [Op.not]: null },
        },
        {
          imported: {
            targetPopulations: {
              [Op.like]: '%Affected by Child Welfare involvement%',
            },
          },
        },
      ],
```

can probably just be

```js
   where: {     
          imported: {
            targetPopulations: {
              [Op.like]: '%Affected by Child Welfare involvement%',
            },
          },
   },    
 ```

Reply:
This has been changed. I guess I was worried that the null value would break the where clause.